packages/engine/Source/Scene/ITwinData.js

```diff
@@ -71,4 +72,63 @@ ITwinData.createTilesetFromIModelId = async function (iModelId, options) {
   return Cesium3DTileset.fromUrl(resource, options);
 };
 
+/**
+ * Create a tileset for the specified reality data id. This function only works
+ * with 3D Tiles meshes and point clouds.
+ *
+ * If the <code>type</code> or <code>rootDocument</code> are not provided this function
+ * will first request the full metadata for the specified reality data to fill these values.
+ *
```

Comment:
For consistency with `createTilesetFromIModelId`.

```suggestion
 *
 * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
```